gocop/storer/sql.go

```diff
@@ -0,0 +1,167 @@
+package storer
```

Comment:
we should use `git mv` before the modfiications so we can track the changeset for these files; It doesn't look like it was able to detect this as the same file from `db.go` with the `add/drop`